### ISSUE TITLE
MatekF722SE stuck on Wizard and The FC rebooting

### ISSUE THREAD

**Issue by achmiral** [FlightOne/FalcoX-Issue-Tracker#174]
**Version**
Version 10.1.0.5368 |10.1 Alpha 18|

**Target**
MatekF722SE

**Describe the bug**
Freeze on Detect VTX Wizard and The FC automatically restart

**To Reproduce**
1. Flash FalcoX Version 10.1.0.5368 |10.1 Alpha 18| to Matek F722SE
2. Plug in the lipo
3. Got freeze on Detect VTX Wizard

**Expected behavior**
After plug in the battery the wizard should detect the RX and the VTX

**Screenshots**
https://youtu.be/vF7LdYhbglc

**Additional context**

**Comment by viktor-k99** [comment]
+1
similar bug

**Comment by PGIII** [comment]
Will look into it this week

**Comment by achmiral** [comment]
Great!! thank you @PGIII

**Comment by viktor-k99** [comment]
Version
Version 10.1.0.5407 |10.1 Alpha 19|

Target
MatekF722SE

RC
TBS nano RX

Describe the bug
After plug in the battery the wizard not found the VTX and start search Rreciver.
The search for RX continues endlessly

To Reproduce
Steps to reproduce the behavior:

Flash FalcoX Version 10.1.0.5407 |10.1 Alpha 19|| to Matek F722SE
Plug in the lipo
Wait for the Wizard to start searching for the receiver

Expected behavior
After plug in the battery the wizard should detect the RX and the VTX

**Comment by viktor-k99** [comment]
@achmiral  what version of TBS nano RX firmware do you have?

**Comment by achmiral** [comment]
@viktor-k99 3.78

**Comment by viktor-k99** [comment]
@achmiral I will downgrade the firmware in the receiver and try again

**Comment by viktor-k99** [comment]
I downgraded the Receiver firmware to 3.78, tried to connect  Reciver to UART1 or UART2. The wizard didn't detect the receiver.

I have flashed the Receiver firmware to version 4.10 and soldered the receiver to UART1. I flashed the FC into Betaflight and checked that Betaflight detects the receiver correctly.

I flashed the FC firmware to FalcoX Alpha Version 10.1.0.5414 | 10.1 Alpha 20 |. Updated configurator to version 10.1.6-alpha.
The wizard does't detect the Reciver on UART1.
